test/Table_Tests/src/In_Memory/Table_Spec.enso

```diff
@@ -193,7 +196,7 @@ spec =
 
             x = Column.from_vector 'a' [1, Nothing, 2, Nothing]
             y = Column.from_vector 'b' [0, 0, 0, 0]
-            r = x.zip y skip_missing=False x-> y-> if x == 1 then x else y
+            r = x.zip y skip_nothing=False x-> y-> if x == 1 then x else y
```

Comment:
Good spot. Can we do a quick scan for where we still have `missing` in the code base as we should align I think.

Reply:
Oh, and the `count_missing` method! I guess that one could be renamed.